gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/reactive/debug/DebugReactorEventListener.java

```diff
@@ -152,6 +158,11 @@ private DebugApi toDebugApi(final io.gravitee.repository.management.model.Event
                 io.gravitee.definition.model.debug.DebugApi.class
             );
 
+            if (null != eventPayload.getProperties()) {
+                decryptProperties(eventPayload.getProperties());
+                eventPayload.setProperties(eventPayload.getProperties());
```

Comment:
This looks redundant for me, you are assigning the same value back into the object

Reply:
You're tight thanks